InstallationGuide.md

```diff
@@ -270,7 +271,12 @@ PuMuKIT node and transcoder node should mount the NAS server folder where all th
 
 [(back to index)](#index)
 
-## 4. F.A.Q.
+## 4. Opencast integration
+
+To add [Opencast](http://www.opencast.org/) support to the PuMuKIT platform use the Opencast Bundle. The Opencast Bundle is included in PuMuKIT but it's not activated by default. To activate and configure, follow the instructions in the guide of the bundle: https://github.com/campusdomar/PuMuKIT2/blob/master/src/Pumukit/OpencastBundle/Resources/doc/README.md
```

Comment:
@rubenrua, I would change:
`To activate and configure, follow the instructions in the guide of the bundle: `
by:
`Follow the instructions in the guide of the Opencast bundle to activate it and configure it:`